interface-definitions/include/auth_psk_id.xml.i

```diff
@@ -0,0 +1,11 @@
+<!-- include start from auth_psk_id.xml.i -->
```

Comment:
We mostly (99%) use hypens in include files. Can you please stick to this unwritten rule?

Reply:
fixed